http/bridge/pom.xml

```diff
@@ -179,17 +179,17 @@
         <dependency>
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
-            <version>2.15.0</version>
+            <version>2.18.0</version>
         </dependency>
         <dependency>
             <groupId>org.apache.felix</groupId>
             <artifactId>org.apache.felix.http.base</artifactId>
-            <version>5.1.8</version>
+            <version>5.1.10</version>
         </dependency>
         <dependency>
             <groupId>org.apache.felix</groupId>
             <artifactId>org.apache.felix.http.wrappers</artifactId>
-            <version>1.0.2</version>
+            <version>1.0.6</version>
```

Comment:
1.1.8 is latest for wrappers.

Reply:
Using 1.1.8 would cause breaking changes from API perspective (build failure). Therefore, I sticked on 1.0.6, the latest compatible one.